doc-content/enterprise-only/integration/dev-studio-import-local-git-repo-proc.adoc

```diff
@@ -1,12 +1,12 @@
 [id='dev-studio-import-local-git-repo-proc']
 = Importing a local Git repository
-You can import a local git repository to use with Red Hat JBoss Developer Studio.
+You can import a local Git repository to use with Red Hat Developer Studio.
 
 .Prerequisite
-Red Hat JBoss Developer Studio is installed.
+Red Hat Developer Studio is installed.
 
 .Procedure
-. In Red Hat JBoss Developer Studio, select the server from the *Server* tab and click the start icon to start the server.
+. In Red Hat Developer Studio, select the server from the *Server* tab and click the start icon to start the server.
 . Select *File* -> *Import* and navigate to the Git folder. Open the Git folder to select *Projects from Git* and click *Next*.
```

Comment:
Make this a new step:
Open the Git folder to select *Projects from Git* and click *Next*.

Reply:
Made this a new step

Reply:
Adopted